<launch>
  <arg name="joydev" default="/dev/input/js0" />

  <node name="joy_core" pkg="joy" type="joy_node">
    <param name="dev" type="string" value="$(arg joydev)" />
  </node>

  <node name="joystick_example" pkg="crane_x7_examples" type="joystick_example.py" required="true" output="screen">
    <param name="button_shutdown_1" value="8" type="int" />
    <param name="button_shutdown_2" value="9" type="int" />
    <param name="button_enable"     value="4" type="int" />
    <param name="axis_position_x"   value="0" type="int" />
    <param name="axis_position_y"   value="1" type="int" />
    <param name="axis_position_z"   value="3" type="int" />
  </node>
</launch>
